plugins/reporters/web-app-template/src/components/SummaryView.js

```diff
@@ -402,8 +402,7 @@ class SummaryView extends React.Component {
                                         )
                                     }
                                     {
-                                        webAppOrtResult.hasIssues()
-                                        && (
+                                        (
```

Comment:
I guess we could drop the surrounding `(` / `)` now an de-indent the whole `<TabPane>` block (i.e. lines 378 - 401).

Reply:
For consistency, IMO we should do the same for rule violations and vulnerabilities above and below.

Reply:
I also thought about this, but was not sure, if this was in scope of the initial ticket. Also, should those all be individual commits?

Reply:
To me both a single commit and separate commits would be fine here, as you prefer.

Reply:
If the rationale is the same for all of there changes (which I believe it is), I'd put all into a single commit. But I'm also ok with either way.